tests/testthat/_snaps/demo/demo.md

```diff
@@ -136,7 +136,7 @@ Let's take a look at the contents:
 ```r
 news <- readLines(usethis::proj_path("NEWS.md"))
 cat(news, sep = "\n")
-## # tea 0.0.0.9000
+## # tea (development version)
```

Comment:
Intended?

🙀 🙀 🙀

Reply:
yes! but after the first bump it becomes an actual version. if you don't want this behavior, we should not create the NEWS file with usethis.

I find the new behavior more logical https://github.com/r-lib/usethis/issues/1708 :sweat_smile: :innocent: